src/jit/compiler.cpp

```diff
@@ -7202,7 +7225,8 @@ void JitTimer::PrintCsvHeader()
 
     CritSecHolder csvLock(s_csvLock);
 
-    if (_waccess(jitTimeLogCsv, 0) == -1)
+    FILE* fp = _wfopen(jitTimeLogCsv, W("r"));
+    if (fp == nullptr)
```

Comment:
> Maybe I should just squash everything when I commit / push the latest changes

Right. It will also make the PR mergeable - it has conflicts in its current state.